src/App.js

```diff
@@ -11,9 +13,10 @@ function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <h1 className="App-title">Application title</h1>
+        <h1 className="App-title">My Timeline</h1>
```

Comment:
Just noting you can use the data in `timelineData` to set the title.

```suggestion
        <h1 className="App-title">{timelineData .person}'s Timeline</h1>
```